woocommerce/woocommerce
# [CYS] Update the neutral color palette to black and white

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Update the Neutral color palette to black and white instead of relying on Tsubaki's color palette.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

### Initial Setup

1. Create a new WooCommerce installation with this version.
2. Ensure Jetpack is installed and your site is connected with JP.
3. Ensure the WooCommerce Beta Tester plugin is installed and activated (available on this monorepo).
4. Ensure the Woo AI plugin is installed and activated (available on this monorepo).
5. Head over to `/wp-admin/tools.php?page=woocommerce-admin-test-helper` and enable `customize-store` feature flag:

<img width="1233" alt="Screenshot 2023-10-22 at 10 32 24" src="https://github.com/woocommerce/woocommerce-blocks/assets/15730971/816867c6-34e5-40a1-a87d-f4a7cda46429">

### Test the new color palette in offline mode

1. Head over to WooCommerce -> Home 
2. Open your browser's inspector and use Chrome's content override [feature](https://developer.chrome.com/docs/devtools/overrides/) to change the content of https://status.openai.com/api/v2/status.json call. Change status.indicator to major.
2. Click the Design with AI button
3. Fill out a business description and proceed
4. Whenever you reach the Assembler, click "Change the color palette" and pick the first one displayed (black and white).
5. Save Your changes and confirm the selected color palette is visible on the front end.

<img width="1233" alt="Screenshot 2023-12-26 at 09 43 02" src="https://github.com/woocommerce/woocommerce/assets/15730971/d8743f1b-72e5-4d11-8022-ccd6b5e27bc8">




### Changelog entry




<details>

#### Significance




#### Type




#### Message 

Update the neutral color palette in the Customize your Store Pattern Assembler from Tsubaki colors to black and white.

#### Comment 

</details>
